genestack_client/file_filters.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, file_type):
         super(TypeFileFilter, self).__init__()
         if not JavaFileTypes.is_file_type(file_type):
             raise GenestackException("Invalid file type")
-        self._dict = {'type': file_type}
+        self._dict.update({'type': file_type})
```

Comment:
You can simplify it, but this is nor required for merge
`self._dict['type'] = file_type`